.storybook/preview.js

```diff
@@ -25,7 +25,7 @@ const guardianViewports = {
   phablet: {
       name: 'phablet',
       styles: {
-          width: '660px',
+          width: '660px', 
```

Comment:
Was it this change that caused the diffs in Chromatic?

Reply:
I have no idea, it seems to have changed focus?? :/ a little concerning